services/dynamic-sidecar/Dockerfile

```diff
@@ -187,7 +187,7 @@ RUN chmod +x services/dynamic-sidecar/docker/*.sh
 HEALTHCHECK \
   --interval=10s \
   --timeout=5s \
-  --start-period=30s \
+  --start-period=60s \
```

Comment:
I would recommend using a larger buffer. We conducted a single test on a project with 300,000 files, where we tested just one command-line operation to change permissions, which took 15 seconds. 
- Firstly, the entire startup process takes longer than 15 seconds.
- Secondly, what if the user has more files? I’d like to avoid hearing complaints about this for a while, so I suggest setting the buffer to at least 3 minutes.